go.mod

```diff
@@ -3,22 +3,26 @@ module github.com/submariner-io/lighthouse
 go 1.13
 
 require (
-	github.com/caddyserver/caddy v1.0.5
-	github.com/coredns/caddy v1.1.1
-	github.com/coredns/coredns v1.9.1
+	cloud.google.com/go/compute v1.3.0 // indirect
+	github.com/evanphx/json-patch v4.12.0+incompatible // indirect
+	github.com/google/uuid v1.3.0 // indirect
 	github.com/kelseyhightower/envconfig v1.4.0
-	github.com/miekg/dns v1.1.48
 	github.com/onsi/ginkgo v1.16.5
 	github.com/onsi/gomega v1.19.0
 	github.com/pkg/errors v0.9.1
 	github.com/prometheus/client_golang v1.12.1
 	github.com/submariner-io/admiral v0.12.0-m3.0.20220223161649-65232cddf5c0
 	github.com/submariner-io/shipyard v0.12.0-m3.0.20220421170648-adf7571e55ed
 	github.com/uw-labs/lichen v0.1.5
+	golang.org/x/sys v0.0.0-20220209214540-3681064d5158 // indirect
+	golang.org/x/time v0.0.0-20211116232009-f0f3c7e86c11 // indirect
+	gopkg.in/check.v1 v1.0.0-20201130134442-10cb98267c6c // indirect
 	k8s.io/api v0.23.4
 	k8s.io/apimachinery v0.23.4
 	k8s.io/client-go v11.0.0+incompatible
```

Comment:
Can we now change these to v0.21.11?

Reply:
All the changes in the base `go.mod` were automated, I’d rather not add manual changes. `go mod tidy` always bumps `coredns/go.mod`’s `k8s.io` dependencies to v0.23.4, so I can’t specify v0.21.11 there either.

Reply:
Done, both `go.mod`s can be simplified quite a bit, but this should end up requiring Go 1.17 (see https://github.com/submariner-io/shipyard/pull/775).